packages/desktop-client/src/components/reports/graphs/DonutGraph.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { VictoryLabel, VictoryPie, VictoryTooltip } from 'victory';
+
+//import theme from '../chart-theme';
+
+function DonutGraph({ style, start, end, graphData }) {
+  return (
+    graphData && (
+      <svg viewBox="0 0 500 325">
+        <VictoryPie
+          standalone={false}
+          width={350}
+          height={350}
+          labelPosition="centroid"
+          labelPlacement="parallel"
+          labelRadius={100}
+          //padding={50}
+          data={graphData}
+          //theme={theme}
+          colorScale="qualitative"
+          innerRadius={80}
+          labelComponent={<VictoryLabel textAnchor="middle" />}
+          labels={({ datum }) => `${datum.x}`}
```

Comment:
I’m not sure how fixable this is but it would be great to flip the text to white for dark colors (or just make sure all of the colors are light enough to have good contrast):

<img width="141" alt="Screenshot_2023-03-09 09 40 58" src="https://user-images.githubusercontent.com/25517624/224058878-f6b5ec9c-2cd4-4fb7-b0d6-4e955c1dbcce.png">

Reply:
Only problem I see with flipping text is that the longer category names spill over onto the white background killing our attempts at contrast.

I've created a "colorScale" using some of the lighter colors from the style.js. I think it solves the contrast issue. Happy to discuss if these are the colors we want to use.

I do forsee an issue if given the right number of categories you would end up with 2 yellows next to eachother. Not sure how to fix this though.